### PR TITLE
Fetch replication status in its own resource

### PR DESCRIPTION
**Pull request by jaymalasinha**
Adding the replication status output to the verification script caused json decode error, so fetch the status in its own resource

Signed-off-by: Jaymala Sinha <jaymala@hashicorp.com>

### CODE REVIEW

**Review by ryancragun — APPROVED** [review]
🚀

File: enos/modules/vault_verify_performance_replication/main.tf [hunk 1]
```diff
@@ -64,6 +64,21 @@ resource "enos_remote_exec" "verify_replication_on_primary" {
   }
 }
 
+resource "enos_remote_exec" "get_primary_replication_status" {
```
Comment (mikebaum) [inline enos/modules/vault_verify_performance_replication/main.tf:67]: Do we not need to put explicit depends_on clauses on this resource and the next to ensure that they run after the verify resource has completed?

**Review by mikebaum — COMMENTED** [review]
just one question, but looks good.

File: enos/modules/vault_verify_performance_replication/main.tf (hunk 1 above)
Comment (jaymalasinha) [reply enos/modules/vault_verify_performance_replication/main.tf:67]: humm.. good point @mikebaum .  But we do want this resource to be run even when the previous one fails, but also don't want it to run before or in parallel.  probably that was the reason I originally called it inside the verification script.  wondering if I can call the inline after the script in the verify resource

Comment (jaymalasinha) [reply enos/modules/vault_verify_performance_replication/main.tf]: To resolve this issue, I renamed the script to correctly identify it with its purpose `get-replication-status`

**Review by mikebaum — APPROVED** [review]
🏆

**Pull request merged**